openapi.yaml

```
@@ -73,28 +75,66 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/ValidateDocument'
+              $ref: "#/components/schemas/ValidateDocument"
       responses:
-        '204':
+        "204":
           description: The given AsyncAPI document is valid.
-        '400':
+        "400":
           description: The given AsyncAPI document is not valid.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Problem'
-        '422':
+                $ref: "#/components/schemas/Problem"
+        "422":
           description: The given AsyncAPI document is not valid due to invalid parameters in the request.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Problem'
+                $ref: "#/components/schemas/Problem"
         default:
           description: Unexpected problem.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Problem'
+                $ref: "#/components/schemas/Problem"
+  /bundle:
+    post:
+      summary: Bundle the given AsyncAPI document.
```

```suggestion
      summary: Bundle the given AsyncAPI document(s).
```

We can bundle single file or merge 2+.